install.sh

```diff
@@ -210,7 +210,16 @@ __install_bro() {
 			;;
 	esac
 	__install_packages bro broctl
-	chmod 2755 /opt/bro/logs
+	if [ -d /opt/bro/logs/ ]; then
+		chmod 2755 /opt/bro/logs
+	elif [ -d /var/log/bro/ ]; then
+		mkdir -p /opt/bro/logs/
```

Comment:
Could you add some comments as to why this is needed please? e.g. Something like (if accurate) `# RHEL and Ubuntu OS bro packages store logs in /var/log/bro. Link them to /opt/bro/logs to standardize the location.`

Reply:
Agreed, will add a comment to that effect.